mockbee-test/ecom-store/src/components/ProductCard/ProductCard.jsx

```diff
@@ -28,8 +35,22 @@ const ProductCard = ({ imgUrl, isTrending, title, productCategory,
             Express Delivery
           </small>
         )}
-        <button className="badge-close cursor absolute fav-outline d-flex align-center justify-center">
-          <span className="material-icons-outlined">favorite</span>
+        <button
+          className="badge-close cursor absolute fav-outline d-flex align-center justify-center"
+          onClick={() =>
+            productDispatch({
+              type: "WISHLIST",
+              payload: product
+            })
+          }
+        >
+          {myWishlist.some((p) => p.id === product.id) ? (
```

Comment:
make this a utility function so that you can use that same function some other place